src/unleash.ts

```diff
@@ -87,6 +91,14 @@ export class Unleash extends EventEmitter {
   }: UnleashConfig) {
     super();
 
+    Unleash.instanceCount++;
+
+    if(Unleash.instanceCount > 10) {
+      console.warn('The unleash SDK has been initialized more than 10 times')
+    }
+
```

Comment:
Regarding whether we should warn or not. I'm into the idea, but maybe make it configurable? For instance, add a `disableInstanceCountWarning` variable that can turn it off or something? 

Also, should it be `console.warn` or a configurable logger? I don't know how you usually do that in Node, but I'd expect to be able to redirect warnings to where I'd want them, I think. Also, if it's a configurable logger (or just a `logWarning: (warning: string) => void` function, then you could pass `() => {}` as an argument and disable the logging that way 🤷🏼

Reply:
Good point. We currently do not support a log abstraction, and I think it would be more work to get that in than I am willing to do for this change. We do have the event emitter, but the purpose here is to want users of the SDK that something is off. 

A suitable middle-ground would be to add the option to disable the warning entirely, and log it as an error to the event emitter, which default will console.error it, and the user can override that by registering an on("error") handler.

Reply:
> I think it would be more work to get that in than I am willing to do for this change

Aye, I agree with this.

> A suitable middle-ground would be to add the option to disable the warning entirely, and log it as an error to the event emitter, which default will console.error it, and the user can override that by registering an on("error") handler.

Just to make sure I understand this: does it only log to the event emitter if you disable the warning, or does it log to the event emitter regardless? And do we have support "warning" events? This isn't an error, so logging it as such feels a little icky to me.

Reply:
> Just to make sure I understand this: does it only log to the event emitter if you disable the warning

No, the plan is then to skip the error entirely,